.github/workflows/golangci-lint.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
           go-version: 1.21.x
+          cache: false
```

Comment:
There is a double cache with golangci-lint’s .

Reply:
This file is managed upstream in `prometheus/prometheus`.